data/sabrinalzbh.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "Sabrina Chacon",
+  "type": "personal",
+  "bio": "Freelancer | Coffee Lover | Code Lover",
+  "tags": ["Open Source", "DevRel", "Blogging"],
+  "links": [
+    {
+      "group": "Socials Primary",
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Dev_HopeFuture",
+      "icon": "FaTwitter"
+    },
+    {
+      "group": "Socials Secondary",
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/dev_hopefuture/",
+      "icon": "FaInstagram"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sabrinalzbh/",
+      "icon": "FaGithub"
+    }
+  ]
+},
```

the links in the JSON file have been closed too early
```suggestion
    },
```

---

> the links in the JSON file have been closed too early

Hi Sara ☺️, I think I accepted the suggestion. Not so sure if I did it correctly because I dared to use my phone. I hope it worked.